example/example.py

```diff
@@ -6,7 +6,7 @@
 schema = Schema([
     Column('Given Name', [LeadingWhitespaceValidation(), TrailingWhitespaceValidation()]),
     Column('Family Name', [LeadingWhitespaceValidation(), TrailingWhitespaceValidation()]),
-    Column('Age', [InRangeValidation(0, 120)]),
+    Column('Age', [InListValidation(['unspecified']) | InRangeValidation(0, 120)]),
```

Comment:
Why do you think this needs to be in the example? It is possible to use `InRangeValidation` without `InListValidation`. I want to keep a simple example.

Reply:
You're right, the idea was to show you can do it, but since this is the basic examples this is not te correct place.
I think it's not necessary to add it on the documentation since 1. no one asked before, 2. the library will handle the case automatically.